pkg/consistenthash/maglev/maglev_test.go

```diff
@@ -0,0 +1,176 @@
+package maglev
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"unsafe"
+
+	"github.com/cilium/ebpf"
+	"github.com/stretchr/testify/suite"
```

Comment:
Add a blank line to separate Kmesh pkgs from other pkgs.

Reply:
done.